Products/Zuul/facades/devicefacade.py

```diff
@@ -770,9 +770,13 @@ def getBoundTemplates(self, uid):
 
     def _getBoundTemplates(self, uid, isBound):
         obj = self._getObject(uid)
-        for template in obj.getAvailableTemplates():
-            if (template.id in obj.zDeviceTemplates) == isBound:
-                yield template
+        templates = (template for template in obj.getAvailableTemplates()
+             if (template.id in obj.zDeviceTemplates) == isBound)
```

Comment:
May I suggest a possibly easier-to-read formatting of this statement?
```
templates = (
    template
    for template in obj.getAvailableTemplates()
    if (template.id in obj.zDeviceTemplates) == isBound
)
```
Comprehensions like this have a lot going on and I've found the above formatting to have a lower cognitive load when reading it.

Reply:
Yep, why not.